leaderboardFeature/src/main/java/com/example/leaderboardfeature/ui/LeaderboardFragment.kt

```diff
@@ -0,0 +1,79 @@
+package com.example.leaderboardfeature.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.fragment.app.Fragment
+import com.example.leaderboardfeature.model.DataProvider
+import com.example.leaderboardfeature.R.string
+import com.example.leaderboardfeature.databinding.FragmentLeaderboardBinding
+
+class LeaderboardFragment : Fragment() {
+  private lateinit var binding: FragmentLeaderboardBinding
```

Comment:
use backing nullable field _binding, set _binding to `null` in `onDestroyView` [see more] (https://developer.android.com/topic/libraries/view-binding#fragments)

Reply:
current approach leads to memory leaks